administration/src/cards/extensions/BirthdayExtension.tsx

```diff
@@ -45,10 +60,16 @@ const BirthdayForm = ({
         onChange={changeBirthday}
         onClear={() => setValue({ birthday: null })}
         isValid={isValid || !showErrorMessage}
+        isHint={isBirthdayHintEnabled()}
         maxDate={new Date()}
         disableFuture
       />
-      {showErrorMessage && <FormErrorMessage errorMessage={getErrorMessage()} />}
+      {showErrorMessage && (
+        <FormErrorMessage
```

Comment:
If we want to use this for a hint(info) and error we could also use a more neutral name like `Alert` which is also using mui for naming and the severity can be different. The severity the can define the message color